security_and_privacy_self_review.md

```diff
@@ -0,0 +1,114 @@
+# Security & privacy self-review for site-initiated mirroring
+
+The questions are taken from
+https://www.w3.org/TR/security-privacy-questionnaire.
+
+## Authors
+
+- Takumi Fujimoto [@takumif](https://github.com/takumif) |
+  [<takumif@google.com>](mailto:takumif@google.com)
+
+## 2.1 What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+A website would be able to query (get events fired containing a boolean) whether
+the user has devices on the local network that support being a receiver for
+site-initiated mirroring (e.g a Chromecast device). Note that it could already
+query for devices that support existing uses of Presentation API (launching a
+web app by URL), and those sets of devices overlap for the most part.
```

Comment:
Don't they overlap completely?  So the "mirroring" bit is not any new information.

Reply:
In theory there can be devices that support 2UA but not mirroring, or vice versa, and if the user has those devices then the website can learn more by querying both separately.

Reply:
Yes, but there's no actual product that implements one or the other but not both.  Without knowing the product situation, the reader will assume that new information is being provided, when it's not in practice.

Reply:
Yeah I think you're right; updated.